### ISSUE TITLE
Feature Request: Multiple Accounts

### ISSUE THREAD

**Issue by benyanke** [digitalocean/doctl#266]
I am not sure what it would look like for the command syntax, but multiple accounts would be an excellent feature.

Personally, I set up all my clients on their own DO account, for billing purposes, so doctl is borderline unusable for quick tasks.

I see two simple options for the syntax and implmenetation (there may be more):

1) add another auth command with shortcuts - `doctl auth switch [accountnickname]` switches the context to the other account. This is backward compatible. 

2) The breaking change option would be specifying the account nickname in every command. Also more stateless though.

**Comment by xmudrii** [comment]
@benyanke In the meanwhile.. there should be the `--access-token` flag which can help you.
However, that would require you to store tokens somewhere accessible (yet secure). Maybe you can use environmental variables as a workaround solution.

**Comment by eRadical** [comment]
On AWS the aws-cli uses a simple file to store only credentials:
```
[eRadical@hp-g3-acasa ~]$ cat ~/.aws/credentials 

[default]
aws_access_key_id = default_AK
aws_secret_access_key = default_SAK

[demo1]
aws_access_key_id = X1
aws_secret_access_key = Y1

[demo2]
aws_access_key_id = X2
aws_secret_access_key = Y2
```

Then in command line you can do:

```
aws --profile=demo2 [....]
```

If you don't provide a profile it goes to the default one if it exists otherwise it errors out.

It would be lovely to do it like that... as I read the credentials file from ansible and use the keys from those files. This way credentials are in one place only and used by  doctl and ansible code.

**Comment by ReggaePanda** [comment]
a `doctl auth switch [accountnickname]` will be a nice feature (also for teams) is this on the roadmap or can I place somewhere a feature request?

**Comment by mauricio** [comment]
It is, we should have it available soon :)

**Comment by jrm780** [comment]
This feature is now available in doctl v1.8.0 :) thanks @kamaln7 !
